### DISCUSSION TITLE
Using http instead of https locally with localized domains

### DISCUSSION THREAD

**Discussion by eiskalteschatten** [vercel/next.js#23251] [Help]
I have a Next.js application that uses two different domains for different languages. English uses .com and German uses .de. When I upgraded to Next.js 10, I rewrote the entire mechanism to use the built-in localization feature that supports multiple domains out of the box which is absolutely awesome and works like a charm in production.

However, I have had problems locally with development since then. When I click on a link, it automatically redirects to `https` which I am not using for development which of course leads to the browser not being able to connect to the page. I use the following domains for local development which, of course, are set up in `/etc/hosts`:

127.0.0.1       en.localhost
127.0.0.1       de.localhost

Of course, I can access the application over localhost without the subdomain and the links work perfectly fine, but then I have no way to distinguish between the languages for development. Anyone have any suggestions on how to work around this? Is this a bug that I should report?

**Comment by MushuPlatypus** [comment]
I'm having the same problem so if you find a solution please post it here :)

**Reply by eiskalteschatten** [reply]
Unfortunately, I still don't have a solution yet... I always have to change `https` to `http` every time I change the page. It's immensely frustrating.

**Comment by eiskalteschatten** [comment]
Since nobody seems interested in the issue here, I just filed a bug report: https://github.com/vercel/next.js/issues/24991
